modules/commandModule/commandFns.py

```diff
@@ -31,4 +72,44 @@ def write_pigo(newPigo):
 	return
 #if nothing has changed, return the dictionary that was passed in
     else:
-        return
\ No newline at end of file
+        return
+
+def read_pogi(POGI_DIR="") -> tuple:
+
+    """
+    Reads all data currently in the POGI file and returns as a dict
+
+    Parameters
+    ----------
+    POGI_DIR: str
+        directory to pogi json file; default == ""
+
+    Returns
+    -------
+    tuple:
+        contains changed flag and pogi dict; i.e. (true, pogi) --> pogi is different from latest_pogi.json
+    """
+
+    command = CommandModule(pigoFileDirectory=PIGO_DIR, pogiFileDirectory=POGI_DIR)
+
+    # store all current POGI data into a dict
+    # note: the keys included in the dict are those which are currently valid and working in commandModule
+    #       (some fields are defined in the wiki but are currently commented out)
+    pogiData =  {
+    'currentAltitude' : command.get_current_altitude(),
+    'currentAirspeed' : command.get_current_airspeed(),
+    'isLanded' : command.get_is_landed(),
+    'eulerAnglesOfCamera' : command.get_euler_angles_of_camera(),
+    'eulerAnglesOfPlane' : command.get_euler_angles_of_plane(),
+    'gpsCoordinates' : command.get_gps_coordinates(),
+    }
+
+    latestJsonDirectory = ".\\latest_pogi.json"
```

Comment:
Are you trying to access a path from the current directory? If so, use `os.path.join(os.getcwd(), "latest_pogi.json")`. Also note that when the program runs, os.getcwd() will always be the root folder where main.py is running, so you may need to navigate to an additional folder.